ArchiSteamFarm/Json/Steam.cs

```diff
@@ -261,7 +261,7 @@ public sealed class Tag {
 
 				internal Tag(string identifier, string value) {
 					Identifier = !string.IsNullOrEmpty(identifier) ? identifier : throw new ArgumentNullException(nameof(identifier));
-					Value = !string.IsNullOrEmpty(value) ? value : throw new ArgumentNullException(nameof(value));
+					Value = value;
```

Comment:
That's wrong, it doesn't throw on `null`. Should be `Value = value ?? throw new ArgumentNullException(nameof(value));`

Reply:
C# would be a better language if nullability for references was actually checked by the runtime.

Reply:
Yes 😎